internal/schema/ize-spec.json

```diff
@@ -3,31 +3,40 @@
     "type": "object",
     "properties": {
         "aws_profile": {
-            "type": "string"
+            "type": "string",
+            "description": "AWS profile"
         },
         "aws_region": {
-            "type": "string"
+            "type": "string",
+            "description": "AWS region"
         },
         "config_file": {
-            "type": "string"
+            "type": "string",
+            "description": "path to config file"
         },
         "env": {
-            "type": "string"
+            "type": "string",
+            "description": "environment"
         },
         "env_dir": {
-            "type": "string"
+            "type": "string",
+            "description": "environment directory"
         },
         "home": {
-            "type": "string"
+            "type": "string",
+            "description": "user home directory"
         },
         "infra_dir": {
-            "type": "string"
+            "type": "string",
+            "description": "infrastructure directory"
         },
         "log_level": {
-            "type": "string"
+            "type": "string",
+            "description": "log level"
         },
         "namespace": {
-            "type": "string"
+            "type": "string",
+            "description": "namespace"
```

Comment:
```suggestion
            "description": "(required) Namespace of the project can be specified here. It is used as a base for all naming. It can be overriden by NAMESPACE env var or --namespace flag."
```